### ISSUE TITLE
Not getting messages from old subscribed threads

### ISSUE THREAD

**Issue by virus-found** [wee-slack/wee-slack#754]
Hi.

I wonder if it's a bug or still not implemented:
I am not receiving messages in wee-slack that come from threads (which I subscribed to via official client), started long time ago. I get such messages from Slack email digest.

And I wonder, if it's possible to subscribe to a thread in wee-slack?

**Comment by trygveaa** [comment]
> I am not receiving messages in wee-slack that come from threads (which I subscribed to via official client), started long time ago. I get such messages from Slack email digest.

Yeah, I've experienced this too. The issue is that we only process thread messages where we can find the parent. So if the parent is older than the message history wee-slack has fetched, the thread message is ignored.

The way to fix this would be to generate the message hash if we can't find the parent and call the API to get the parent message and the other messages in the thread.

Related issue: #619

> And I wonder, if it's possible to subscribe to a thread in wee-slack?

Not currently. I'm not sure if there's an API method available for this or not.

> I wonder if it's a bug or still not implemented:

It's not implemented, but from a users perspective it might as well be a bug, so I labelled it as such.
